### PR TITLE
Prekeys investigation: test

### PR DESCRIPTION
**Pull request by jschaul**
## Checklist

 - [x] Add a new entry in an appropriate subdirectory of `changelog.d`
 - [x] Read and follow the [PR guidelines](https://docs.wire.com/developer/developer/pr-guidelines.html)

### CODE REVIEW

File: deploy/dockerephemeral/docker-compose.yaml [hunk 1]
```diff
@@ -161,7 +161,7 @@ services:
   elasticsearch:
     container_name: demo_wire_elasticsearch
     #image: elasticsearch:5.6
-    image: julialongtin/elasticsearch:0.0.9-amd64
+    image: elasticsearch:6.8.23
```
Comment (fisx) [inline deploy/dockerephemeral/docker-compose.yaml:164]: unrelated, but fine.  :)

File: services/brig/src/Brig/App.hs [hunk 2]
```diff
@@ -231,8 +231,12 @@ newEnv o = do
     SqsQueue q -> SqsQueue <$> AWS.getQueueUrl (aws ^. AWS.amazonkaEnv) q
   mSFTEnv <- mapM Calling.mkSFTEnv $ Opt.sft o
   prekeyLocalLock <- case Opt.randomPrekeys o of
-    Just True -> Just <$> newMVar ()
-    _ -> pure Nothing
+    Just True -> do
+      Log.warn lgr $ Log.msg (Log.val "randomPrekeys: active")
+      Just <$> newMVar ()
+    _ -> do
+      Log.warn lgr $ Log.msg (Log.val "randomPrekeys: not active; using dynamoDB instead.")
```
Comment (fisx) [inline services/brig/src/Brig/App.hs:238]: why warn, and not info?  (i trust you, just wondering.)

**Review by fisx — APPROVED** [review]

**Comment by fisx** [comment]
look mergeable to me?  if you disagree please reject my review.

File: deploy/dockerephemeral/docker-compose.yaml (hunk 1 above)
Comment (jschaul) [reply deploy/dockerephemeral/docker-compose.yaml]: good point, unrelated, removed from this PR.

File: services/brig/src/Brig/App.hs (hunk 2 above)
Comment (jschaul) [reply services/brig/src/Brig/App.hs]: no good reason, I changed this to Info, that's better, thanks!

**Pull request merged**